Session.js

```diff
@@ -44,7 +53,23 @@ export default class Session {
     }
   }
 
+  /**
+   * Sets a timeout based on the cookie's maxAge that will emit an
+   *   expire event.
+   *
+   * @param {object} options - Options to use.
+   * @param {number} options.timeout - The timeout.
+   *
+   * @returns {undefined} Just emits an event.
+  */
+  setTimeout({timeout}) {
```

Comment:
I don't think this should be a public method on the session.

Reply:
Addressed here: https://github.com/digitalbazaar/bedrock-web-session/commit/3ba295a12e0a1309909774da1a5da905d965e1ef